src/simulation/openables/HAMChests.ts

```diff
@@ -0,0 +1,61 @@
+import LootTable from '../../structures/LootTable';
```

Comment:
name this file something like `HAMStoreRoomChests` to be more descriptive of what they are at a glance